### ISSUE TITLE
Enterobase Downloads Broken

### ISSUE THREAD

**Issue by lowandrew** [WGS-TB/MentaLiST#81]
Trying to use the `download_enterobase` command with mentalist v0.2.3 looks to be broken - it seems that Enterobase has changed up their underlying file structure.

For example, when trying to get cgMLST for _E. coli_, mentalist is trying to curl from  `https://enterobase.warwick.ac.uk/schemes/ESCwgMLST.cgMLSTv1`, but the files are now stored at `http://enterobase.warwick.ac.uk/schemes/Escherichia.cgMLSTv1`. Salmonella and Yersinia look to be suffering from the same issue.

**Comment by dfornika** [comment]
Thanks for reporting. We've had similar issues in the past when MLST scheme servers change their API. We'll put together a fix as soon as possible.

**Comment by dfornika** [comment]
For our reference here is the schemes listed at [https://enterobase.warwick.ac.uk/schemes/]()

```
Bacillus.rMLST/                                    10-Sep-2018 16:10                   -
Escherichia.Achtman7GeneMLST/                      10-Sep-2018 16:11                   -
Escherichia.Griffiths_MLST/                        10-Sep-2018 16:11                   -
Escherichia.NFVir/                                 10-Sep-2018 16:11                   -
Escherichia.Type3Effectors/                        10-Sep-2018 16:11                   -
Escherichia.cgMLSTv1/                              10-Sep-2018 16:11                   -
Escherichia.rMLST/                                 10-Sep-2018 16:11                   -
Escherichia.wgMLST/                                10-Sep-2018 16:11                   -
Helicobacter.rMLST/                                10-Sep-2018 16:11                   -
Listeria.Pastuer/                                  10-Sep-2018 16:11                   -
Listeria.pasteurcgmlst/                            10-Sep-2018 16:11                   -
Listeria.rMLST/                                    10-Sep-2018 16:11                   -
Moraxella.Achtman7GeneMLST/                        10-Sep-2018 16:11                   -
Moraxella.rMLST/                                   10-Sep-2018 16:11                   -
Mycobacterium.rMLST/                               10-Sep-2018 16:11                   -
Neisseria.rMLST/                                   10-Sep-2018 16:11                   -
Photorhabdus.cgMLSTv1/                             10-Sep-2018 16:18                   -
Photorhabdus.rMLST/                                10-Sep-2018 16:11                   -
Salmonella.Achtman7GeneMLST/                       10-Sep-2018 16:14                   -
Salmonella.cgMLSTv2/                               10-Sep-2018 16:11                   -
Salmonella.rMLST/                                  10-Sep-2018 16:11                   -
Salmonella.wgMLST/                                 10-Sep-2018 16:11                   -
Vibrio.rMLST/                                      10-Sep-2018 16:11                   -
Yersinia.Achtman7GeneMLST/                         08-Oct-2018 17:54                   -
Yersinia.McNally/                                  10-Sep-2018 16:11                   -
Yersinia.cgMLSTv1/                                 08-Oct-2018 17:44                   -
Yersinia.rMLST/                                    10-Sep-2018 16:11                   -
Yersinia.wgMLST/                                   08-Oct-2018 17:49                   -
clostridium.Griffiths/                             10-Sep-2018 16:11                   -
clostridium.Griffiths_MLST/                        10-Sep-2018 16:11                   -
clostridium.cgMLSTv1/                              10-Sep-2018 16:11                   -
clostridium.rMLST/                                 10-Sep-2018 16:11                   -
clostridium.wgMLST/                                10-Sep-2018 16:11                   -
daily_hierBG.bash/                                 10-Sep-2018 16:11                   -
daily_hierBG.bash2/                                10-Sep-2018 16:11                   -
klebsiella.pasteur7gene/                           10-Sep-2018 16:11                   -
klebsiella.pasteurcgmlst/                          10-Sep-2018 16:11                   -
klebsiella.rMLST/                                  10-Sep-2018 16:11                   -
nohup.out/                                         10-Sep-2018 16:11                   -
```

**Comment by dfornika** [comment]
It looks like we'll need to make the following adjustments in [download_enterobase_scheme()](https://github.com/WGS-TB/MentaLiST/blob/083a0275f6f5186c688d4b6360e901333a03efc4/src/mlst_download_functions.jl#L157):

```
sp = Dict("E"=>"ESC", "S"=>"SAL", "Y"=>"YER")
```

...becomes:

```
sp = Dict("E"=>"Escherichia", "S"=>"Salmonella", "Y"=>"Yersinia")
```

We can't assume that a `v1` scheme is available for these three organisms, since there is only a `v2` scheme available for Salmonella:

```
Escherichia.cgMLSTv1
Salmonella.cgMLSTv2
Yersinia.cgMLSTv1
```

**Comment by dfornika** [comment]
May also need to update the scheme files in https://github.com/WGS-TB/MentaLiST/tree/master/scripts

**Comment by dfornika** [comment]
@pedrofeijao Will you have some time to patch this?

**Comment by pedrofeijao** [comment]
I could take a look at it by Friday.

**Comment by pedrofeijao** [comment]
Fixed by commit 8adcc4c859aab42776b6c6c37f869de44aa43b08.
